docs/csharp/programming-guide/classes-and-structs/toc.yml

```
@@ -72,24 +75,22 @@
         href: how-to-know-the-difference-passing-a-struct-and-passing-a-class-to-a-method.md
     - name: Implicitly Typed Local Variables
       href: implicitly-typed-local-variables.md
-      items:
-      - name: "How to: Use Implicitly Typed Local Variables and Arrays in a Query Expression"
-        href: how-to-use-implicitly-typed-local-variables-and-arrays-in-a-query-expression.md
+    - name: "How to: Use Implicitly Typed Local Variables and Arrays in a Query Expression"
+      href: how-to-use-implicitly-typed-local-variables-and-arrays-in-a-query-expression.md
     - name: Extension Methods
       href: extension-methods.md
-      items:
-      - name: "How to: Implement and Call a Custom Extension Method"
-        href: how-to-implement-and-call-a-custom-extension-method.md
-      - name: "How to: Create a New Method for an Enumeration"
-        href: how-to-create-a-new-method-for-an-enumeration.md
+    - name: "How to: Implement and Call a Custom Extension Method"
+      href: how-to-implement-and-call-a-custom-extension-method.md
+    - name: "How to: Create a New Method for an Enumeration"
+      href: how-to-create-a-new-method-for-an-enumeration.md
     - name: Named and Optional Arguments
       href: named-and-optional-arguments.md
-      items:
-      - name: "How to: Use Named and Optional Arguments in Office Programming"
-        href: how-to-use-named-and-optional-arguments-in-office-programming.md
-  - name: Constructors
-    href: constructors.md
+    - name: "How to: Use Named and Optional Arguments in Office Programming"
+      href: how-to-use-named-and-optional-arguments-in-office-programming.md
+  - name: Constructor members
     items:
+    - name: Overview
```

Would this work better for filtering? Otherwise, all entries would be called overview

```suggestion
    - name: Constructors overview
```